src/test/regress/expected/am_query.out

```diff
@@ -119,3 +121,49 @@ INSERT INTO union_second SELECT a, a FROM generate_series(11, 15) a;
 (10 rows)
 
 DROP TABLE union_first, union_second;
+-- https://github.com/citusdata/citus/issues/4600
```

Comment:
I do like the github reference here. It provides a lot of context on how this testsuite came to be.